packages/grid/_modules_/grid/components/virtualization/GridVirtualScrollerContent.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import { styled } from '@mui/material/styles';
+import { unstable_composeClasses as composeClasses } from '@mui/material';
+import { useGridRootProps } from '../../hooks/utils/useGridRootProps';
+import { getDataGridUtilityClass } from '../../gridClasses';
+import { GridComponentProps } from '../../GridComponentProps';
+
+type OwnerState = { classes: GridComponentProps['classes'] };
+
+const useUtilityClasses = (ownerState: OwnerState) => {
+  const { classes } = ownerState;
+
+  const slots = {
+    root: ['virtualScrollerContent'],
+  };
+
+  return composeClasses(slots, getDataGridUtilityClass, classes);
+};
+
+const VirtualScrollerContentRoot = styled('div', {
+  name: 'MuiDataGrid',
+  slot: 'VirtualScrollerContent',
+  overridesResolver: (props, styles) => styles.virtualScrollerContent,
```

Comment:
On HEAD:
https://github.com/mui-org/material-ui-x/blob/cca95ce54d26ec5a4c5d491daf069212480010b9/packages/grid/_modules_/grid/components/GridVirtualScroller.tsx#L55 

This doesn't match the docs:

![image](https://user-images.githubusercontent.com/42154031/139968721-174acf28-0f9b-41bb-ac41-dffd0f49c18a.png)